### PR TITLE
resource-metering: add some metrics

### LINKED ISSUE #11436: Resource metering lacks of report metrics
### What did you expect?
Add more metrics.
* tikv_resource_metering_stat_task_count
* tikv_resource_metering_report_duration_seconds
* tikv_resource_metering_report_data_count
* tikv_resource_metering_ignored_data

### PR DESCRIPTION
**Pull request by zhongzc**
Signed-off-by: Zhenchi <zhongzc_arch@outlook.com>

close #11436

### What problem does this PR solve?

Add some metrics

### Release note 

```release-note
none
```

### CODE REVIEW

**Comment by zhongzc** [comment]
cc @mornyx @breeswish @crazycs520

**Review by crazycs520 — APPROVED** [review]

File: components/resource_metering/src/metrics.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use lazy_static::*;
+use prometheus::*;
+
+lazy_static! {
+    pub static ref STAT_TASK_COUNT: IntCounter = register_int_counter!(
+        "tikv_resource_metering_stat_task_count",
+        "Counter of stat_task call"
```
Comment (breeswish) [inline components/resource_metering/src/metrics.rs:9]: Can we make this description more readable?

**Review by breeswish — APPROVED** [review]
The rest LGTM

**Review by breeswish — APPROVED** [review]

**Review by iosmanthus — APPROVED** [review]
LGTM

**Pull request merged**